spec/factories/state_file_md_intakes.rb

```diff
@@ -83,6 +87,8 @@
     primary_middle_initial { "A" }
     primary_last_name { "Lando" }
 
+    eligibility_filing_status_mfj { "unfilled" }
```

Comment:
is this necessary, given that "unfilled" is the default?

Reply:
that's true, it's not strictly necessary, updated